src/opm/parser/eclipse/EclipseState/InitConfig/FoamConfig.cpp

```diff
@@ -29,6 +29,11 @@ namespace Opm
 
 // FoamData member functions.
 
+FoamData::FoamData()
+    : FoamData(0.0, 0.0, 1e-20, true, 1.0)
```

Comment:
The default values can - somewhat paintfully - be fetched from the generated code:
```
#include <opm/patrser/Parserkeywords/F.hpp>

FoamData( 0,
          ParserKeywords::FOAMFSC::EXPONENT::defaultValue, 
          ParserKeywords::FOAMFSC::MIN_SURF_CONC::defaultValue, 
          ...)
```

Reply:
okay. not sure if it's worth it since the default constructed guys live for a very short time, but in case somebody else end up using the default constructor i will fix it i guess.